### PR TITLE
moved file

### PR DESCRIPTION
**Pull request by jherzberg**

### CODE REVIEW

File: lib/tagnews/geoloc/geocode_list.py [hunk 1]
```diff
@@ -20,11 +20,11 @@ def multi_option_lat_longs(lst, provider='arcgis'):
...
     for addr_str in lst:
+        time.sleep(1)
         g = getattr(geocoder, provider)(addr_str)
         if g.latlng is None:
             time.sleep(.5)
             lats_lons.extend(lat_longs_from_geo_strings([addr_str], provider))
         else:
             lats_lons.append(g.latlng)
     return lats_lons
-
```
Comment (kbrose) [inline lib/tagnews/geoloc/geocode_list.py]: Could you add a newline at the end of the file?

**Comment by kbrose** [comment]
Could you delete the file in the old location too?

`git rm path/to/old/file`

**Comment by jherzberg** [comment]
I think I handled both those things.

**Comment by kbrose** [comment]
Looks good!

**Pull request merged**